boxcli/container.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package boxcli
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"go.jetpack.io/devbox"
+)
+
+type containerCmdFlags struct {
+	config configFlags
+}
+
+func ContainerCmd() *cobra.Command {
+
+	command := &cobra.Command{
+		Use:    "container",
+		Hidden: true,
+		Short:  "Contains interactions with OSI containers. Run 'devbox container -h' for a list of available commands.",
+		Args:   cobra.MaximumNArgs(1),
+	}
+
+	command.AddCommand(exportCmd())
+
+	return command
+}
+
+func exportCmd() *cobra.Command {
+	flags := containerCmdFlags{}
+	command := &cobra.Command{
+		Use:   "export",
+		Short: "Generate Dockerfile to build a container.",
```

Comment:
`build a dev container` - it is only for development purposes, not something people can use in production? Also, it seems like it is very vscode specific